### DISCUSSION TITLE
post-interfaces/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#41] [Announcements]
# post-interfaces/

March.

https://www.supergoodcode.com/post-interfaces/

**Comment by NikitaPuzyryov** [comment]
> No, it’s not Half Life 3 / Portal 3 / L4D3.

Oh it's definitely Half Life 3 / Portal 3 / L4D3.

**Comment by ksthiele** [comment]
> No, it’s not Half Life 3 / Portal 3 / L4D3.

Ah so its TF3 ;)

**Comment by rtobar** [comment]
```
supergoodchair.com: 
DNS_PROBE_FINISHED_NXDOMAIN
```

So sad 😢, for a second I expected it to be there.

**Reply by zmike** [reply]
I thought about it, but a joke should only go so far.
